src/content/docs/recipes/apps/release-your-app.md

```diff
@@ -31,6 +31,30 @@ Discord server / forum. This is not a comprehensive list, but it should help you
 [Ratatui Discord]: https://discord.gg/pMCEU9hNEj
 [Ratatui Forum]: https://forum.ratatui.rs
 
+## Cargo configuration
```

Comment:
I would love to move this to the end of the documentation, maybe under "Optimizations" section and reference it from the Release checklist above. The rationale is to keep the checklist & screenshots under primary focus and provide additional improvements later on. What do you think?

Reply:
I like the idea! Just pushed such a variant ;)